src/util/sss_utf8.c

```diff
@@ -35,55 +35,33 @@
 
 #include "sss_utf8.h"
 
+/* Expects and returns NULL-terminated string;
+ * result must be freed with sss_utf8_free()
+ */
```

Comment:
Hi,

I think this comment fits better to `sss_utf8_tolower()` a couple of line below?

bye,
Sumit

Reply:
Sure. Thanks.